lib/diagnostic.rb

```diff
@@ -17,8 +28,9 @@
 # Finally, assign the modified Person to `Answer.dave` below.
 
 ##
-# your answers here
-Answer.dave = nil
+dave = Person.new("Dave", 32, "Ohio")
+dave()
```

Comment:
You missed the step where you assign a new value, "Somerville", to the new Person's `location` property.